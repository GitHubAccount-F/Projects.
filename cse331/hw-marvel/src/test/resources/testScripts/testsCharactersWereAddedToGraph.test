#Tests to make sure Marvel characters are added to a graph

#Creates graph
LoadGraph graph2 SingleCharacter.csv
LoadGraph graph3 twoCharacters.csv
#Complex example
LoadGraph graph4 directPathNotInSameComics.csv


#Checks if the characters were added to the graph
ListNodes graph2
ListNodes graph3
ListNodes graph4