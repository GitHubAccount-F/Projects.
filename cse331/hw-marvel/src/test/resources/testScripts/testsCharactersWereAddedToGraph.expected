#Tests to make sure Marvel characters are added to a graph

#Creates graph
loaded graph graph2
loaded graph graph3
#Complex example
loaded graph graph4


#Checks if the characters were added to the graph
graph2 contains: SpiderMan
graph3 contains: Carnage Venom
graph4 contains: CaptainAmerica IronMan SpiderMan Thor Venom